src/decorators/Subscription.ts

```diff
@@ -1,11 +1,12 @@
-import { ReturnTypeFunc, AdvancedOptions, SubscriptionFilterFunc } from "./types";
+import { ReturnTypeFunc, AdvancedOptions,
```

Comment:
I wonder why a git pre-commit hook with prettier didn't worked for you 😕 
Maybe run `npx prettier --write "{src,tests,examples}/**/*.{ts,js}"` in your terminal manually.